agent/util-scripts/pbench-tool-data-sink

```diff
@@ -277,6 +380,7 @@ class ToolDataSink(Bottle):
             assert pids["ds"]["hostname"] == self._hostname, f"what? {pids['ds']!r}"
             for tm in pids["tm"]:
                 assert tm["kind"] == "tm", f"what? {tm!r}"
+                self._tms_to_check_prom.append(tm["hostname"])
```

Comment:
We also talked about fetching all the tool meister data at this point, and structuring it so that we can determine when a "collector" needs to be launched for that tool started at `init` time.